packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
+* Mixpanel users will be created if they do not exist on `login`/`signup` in Node (this previously only happened in the browser).
+* Mixpanel is now bundled, and loaded synchronously, so should never fail to load.
```

Comment:
I had the feeling the mixpanel lib is pretty big so I liked the way it was side-loaded without growing our bundle size.

Am I wrong?